release-controller/mock_discourse.py

```diff
@@ -2,12 +2,19 @@
 
 
 class DiscourseClientMock(DiscourseClient):
+    """A mock Discourse client."""
+
     def __init__(self):
+        """Create a new mock client. The actual host needs to be mocked in the test."""
+        self.host = "http://localhost:55555/"
         self.created_topics = []
         self.created_posts = []
         self.api_username = "test"
+        self.api_key = "test_api_key"
+        self.timeout = 10
```

Comment:
this seems to not be used

Reply:
Some tests complained about these missing values, that's why I added this.

```
E       AttributeError: 'DiscourseClientMock' object has no attribute 'api_key'

../python_deps_pydiscourse/site-packages/pydiscourse/client.py:1629: AttributeError
```

Not sure why this suddenly started being required